### DISCUSSION TITLE
Release 17.1.0

### DISCUSSION THREAD

**Discussion by alorle** [IgniteUI/igniteui-angular#13863] [General]
Is there any ETA for v17.1.0?

The release is listed under [CHANGELOG](https://github.com/IgniteUI/igniteui-angular/blob/master/CHANGELOG.md#1710) but there is not entry on the [Update Guide](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/update-guide) nor on the [Github Releases](https://github.com/IgniteUI/igniteui-angular/releases) and the [NPM Releases](https://www.npmjs.com/package/igniteui-angular?activeTab=versions).

Furthermore, the latest version of `igniteui-theming` (v4.0.0) is not compatible with `igniteui-angular` v17.0.0 as pointed out in #13827.

**Comment by georgianastasov** [comment]
Hello @alorle 

There is an estimated time of arrival for Ignite UI Angular version 17.1.0 and it is the middle of this month.

Added version 17.1.0 in the [CHANGELOG](https://github.com/IgniteUI/igniteui-angular/blob/master/CHANGELOG.md#1710) provides information about the new features and general improvements that will be added and available in the new release. After the official release of the version, it will be available in the [Github Releases](https://github.com/IgniteUI/igniteui-angular/releases) and the [NPM Releases](https://www.npmjs.com/package/igniteui-angular?activeTab=versions), and an update section will be added to the [Update Guide](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/update-guide), so you can track from these sources when the new version is officially released.

Furthermore, igniteui-theming is a dependency of the igniteui-angular package and should not be installed separately and manually. It is enough just to install the igniteui-angular package as igniteui-theming will be added as dependencies with the corresponding version about the igniteui-angular version.

**Reply by alorle** [reply]
Thanks for the quick response!!
